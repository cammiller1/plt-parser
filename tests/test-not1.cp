## test not unary operator ##
bool x = True;

if (not x) {
	print("should not go here");
} else {
	print("went to the correct branch");
}