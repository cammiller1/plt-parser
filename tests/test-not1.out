went to the correct branch